runtime/queries/rust/textobjects.scm

```diff
@@ -33,3 +33,5 @@
 (line_comment)+ @comment.around
 
 (block_comment) @comment.around
+
+(block_comment) @comment.block.around
```

Comment:
The textobject captures don't distinguish block vs line or other types of comments currently. Adding a .block. in the middle could be good, but existing code to handle textobject selection may need to be adjusted. @sudormrfbin what do you think?

Actually wondering if we add this to indents.scm instead, done it's kind of related functionality.

Reply:
On second look at indents.scm, I don't think it'll work there easily. Textobjects is probably the best place unless we want a comments.scm specifically for this feature.

Reply:
> but existing code to handle textobject selection may need to be adjusted

Why would it need to be adjusted? It continues using `@comment.around` and that still works.
I'm not sure I understand everything this file does, but I was under the impression that I only added a new capture named `@comment.block.around`.

Reply:
I missed the fact that you added a new capture instead of changing it, my bad! That is effectively adding a whole new capture, which is a good idea. I would maybe drop the 'around' and make up a new one like 'continue'. I'm wondering again in this light of this belongs in the indents.scm queries, since it would be a whole new capture name and wouldn't affect either existing implementations (indent or textobject).

Reply:
In this particular case that second `block_comment` one overrides the first. You could have both captures like

```scm
(block_comment) @comment.around @comment.block.around
```